packages/cosmic-swingset/bin/ag-nchainz

```diff
@@ -232,8 +232,11 @@ start-relayer)
   fi
 
   echo "$ rly config add-paths $PATHCONFIG/"
-  rly config add-paths $PATHCONFIG/
+  while ! rly config add-paths $PATHCONFIG/; do
```

Comment:
What are we waiting for here? i.e. what's likely to change in 3 or 6 or ... seconds?

Reply:
The new `add-paths` tries to verify the path against the chain.  If the chain isn't done booting up (and there's no easy way to tell without trying add-paths), the add-paths will fail.  It will succeed after a few blocks.

Reply:
I'll add a comment.